<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolicyDialog</class>
 <widget class="QDialog" name="PolicyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>840</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>840</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>800</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="helpTextEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>360</width>
     <height>556</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="helpLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Help:</string>
   </property>
   <property name="buddy">
    <cstring>helpTextEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="optionsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Options:</string>
   </property>
  </widget>
  <widget class="QLabel" name="supportedOnLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Supported on:</string>
   </property>
   <property name="buddy">
    <cstring>supportedOnTextEdit</cstring>
   </property>
  </widget>
  <widget class="QTextEdit" name="supportedOnTextEdit">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>130</y>
     <width>411</width>
     <height>45</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="commentTextEdit">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>60</y>
     <width>411</width>
     <height>66</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="commentLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
   <property name="buddy">
    <cstring>commentTextEdit</cstring>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Policy State</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="notConfiguredRadioButton">
      <property name="text">
       <string>Not Configured</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="enabledRadioButton">
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="disabledRadioButton">
      <property name="text">
       <string>Disabled</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Setting</string>
   </property>
  </widget>
  <widget class="QLabel" name="policyNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Example of long policy name</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="optionsScrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>310</width>
     <height>556</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>306</width>
      <height>552</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PolicyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>814</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PolicyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>820</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
